fuzzbench/jobs.py

```diff
@@ -11,25 +11,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Fake jobs."""
+"""Jobs."""
```

Comment:
docstring

Reply:
How about "Defines all required jobs for one experiment."?